src/GameLoop.tsx

```diff
@@ -534,14 +532,37 @@ export function GameLoop() {
     }
 
     return (
-        <BoardStyled>
-            <PlayerComponent
-                isVisible={isPlayerVisible}
-                adjustedCells={playerPhysics.adjustedCells}
-            />
-            <BoardComponent
-                boardCellMatrix={boardPhysics.boardCellMatrix}
-            />
-        </BoardStyled>
+        <>
+            <div style={AppStyle}>
+                <BoardStyled>
+                    <PlayerComponent
+                        isVisible={isPlayerVisible}
+                        adjustedCells={playerPhysics.adjustedCells}
+                    />
+                    <BoardComponent
+                        boardCellMatrix={boardPhysics.boardCellMatrix}
+                    />
+                </BoardStyled>
+                <WordList displayedWords={matchedWords} />
+            </div>
+        </>
     );
 }
+
+const wordStyle = {
+    display: "block",
+    background: "yellow",
+};
+
+const WordList = React.memo(({ displayedWords }) => {
+    return (
+        <div display={"flex"} flex-direction={"column"}>
+            <div>Matched Words ({totalMatchedWords})</div>
```

Comment:
`totalMatchedWords` should be a prop to `WordList`.  Never use globals if you can at all help it.  That said, I think you should just display all the words.

Reply:
Removed